templates/deployment.yaml

```
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "jet-helm-chart.fullname" . }}
+  labels:
+    {{- include "jet-helm-chart.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "jet-helm-chart.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "jet-helm-chart.selectorLabels" . | nindent 8 }}
+    spec:
+      imagePullSecrets:
+        - name: {{ include "jet-helm-chart.fullname" . }}-docker-registry-secret
+      containers:
+        - name: {{ .Chart.Name }}
```

这个为什么直接事 chart name 呢？
```suggestion
        - name: {{ include "jet-helm-chart.image" . }}-jet
```